BUILD.bazel

```diff
@@ -2,10 +2,6 @@ load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
 
 # https://docs.bazel.build/versions/master/be/c-cpp.html#cc_library
 
-# TODO: create multiple libraries and ad them to test cases. T
```

Comment:
This is weird. 
![image](https://user-images.githubusercontent.com/45213495/97928388-3c927400-1d67-11eb-9977-4bcf686d210d.png)
I deleted it because I wrote all libraries for every single test

Reply:
noted that I need to read mroe how to build with bazel